### DISCUSSION TITLE
Governance Model for AspirePress

### DISCUSSION THREAD

**Discussion by sarah-savage** [aspirepress/Governance#3] [Ideas]
Governance is at the heart of any open source project and AspirePress is no exception. This discussion serves as a starting point for creating a governance structure for AspirePress.

# AspirePress Governance Policy (Proposed)

AspirePress exists to serve the WordPress community and its subsidiaries. With that in mind, the AspirePress organization requires effective governance to ensure that the project continues to benefit the open source community for the long term, and can never be bought, sold, or otherwise damaged by the actions of any single individual.

## Board of Directors

AspirePress will have an Board of Directors who is responsible for governing the project, and together they select and appoint an Executive Director who is the manager of AspirePress. The Board of Directors shall consist of no less than three persons, and no more than seven persons.

### Term and Selection

The Board of Directors shall serve for three year terms. The initial board shall consist of five individuals, two of whom shall serve for one year, and two shall serve for two years, and one who will serve for three years. No more than 1/3rd of the board shall be subject to reappointment at any time. 

Every October, the Executive Director shall nominate Board of Directors members to replace the outgoing members. Outgoing members are eligible to remain in their seat as long as the Executive Director renominates them. Upon nomination, the Board of Directors shall approve the nominee with a majority vote. Upon this vote the nominee shall be subject to a 15 day vote of the membership of AspirePress, and a majority vote of those voting is required to approve the nomination. If the nomination is not successful, the Executive Director must nominate a new individual.

If a seat becomes vacant on the Board of Directors at any point, the Executive Director shall nominate a replacement to serve until an election can be held to complete the term. The Board of Directors shall approve the nomination by a majority vote, and the nominee may serve until an election would ordinarily be held. Once the election is held, that member shall serve the remaining term of the previous member.

Members of the Board of Directors serve from January 1 to December 31. The Board of Directors may extend the term of a member for two months if no replacement has been selected by the start of the Board of Directors term.

No member of the Board of Directors shall serve for more than two full terms, and may not be renominated for a third full term. Twelve months must elapse before a Board of Directors member who has reached term limits may be reappointed to the Board of Directors.

### Makeup of the board

In general, AspirePress should work to appoint board members that mirror the user base and community. 

The initial board shall consist of the following makeup:

* 1 seat for hosting providers
* 1 seat for plugin developers
* 2 seats for WordPress developers (agency owners, web designers, core contributors, etc.)
* 1 seat for the community at large (a member of the WordPress community)

If the Board of Directors shrinks to less than the original five, the required seats shall be hosting providers, plugin developers and the community.

### Board Enumerated Powers

The Board of Directors of AsiprePress shall have the ability to conduct the following actions on behalf of AspirePress:

* Hire and fire the Executive Director.
* Review and approve contracts worth more than $500 or that would fundamentally alter the relationship of AspirePress to the outside world (e.g. partnerships)
* Propose new policies and/or procedures, as well as vote upon them for approval.
* Select a Code of Conduct Review Board not made up of Board Members or the Executive Director.
* Provide for allowances to compensate various core contributors and ensure that the underlying entity remains revenue neutral.

### Hiring and Firing of the Executive Director

The Board of Directors is required to select an Executive Director for AspirePress. The Executive Director may not also be a member of the Board of Directors. Hiring an Executive Director shall be conducted in a process defined by the Board, but requires that no less than three members vote in favor of the Executive Director, and at least 2/3rds of the Board of Directors approve the selection.

The Board of Directors may elect to fire the Executive Director with a vote of at least 2/3rds (minimum 3 votes). The Executive Director may only be fired for cause. The Board of Directors may decide not to renew the Executive Director at the end of their term with a majority vote.

### Compensation of the Board of Directors

The Board of Directors shall receive compensation from AspirePress for their services equaling 1% of the net revenues of AspirePress. This shall not, in any circumstance, exceed $10,000 USD in any year. Board members are permitted to share in any distribution of revenues to contributors based on the model they set in place; however such distributions shall be placed in escrow until the Board Member's term expires.

The Board of Directors may elect to raise the $10,000 cap annually by the annual rate of inflation as determined by the United States as of December 1 of the year in which the raise is to be established. The amount may not be further raised beyond the rate of inflation at any time.

### Policy development process

The following governs how a policy is written and published as a policy in AspirePress:

1. Any member of AspirePress may recommend a policy. This is done by creating a personal fork of the governance
   documentation and submitting a pull request with the documentation outlined that the member wishes to add.
2. Upon receiving the recommended policy' all members of AspirePress shall have no less than thirty days to review and
   make comments on the proposed change. If the change is an emergency change this time period may be limited to 5 days.
3. The Board of Directors of AspirePress will meet to discuss the policy. They may take action on the policy in one of four
   ways: they may reject the policy, they may approve the policy, they may request changes to the policy, and they may
   request a vote of the entire membership on the policy. This vote may be binding or non-binding at the discretion of
   the Executive Committee.
4. Upon acceptance of a policy, the policy is merged into `main` as Ratified and becomes official policy of AspirePress.

Changes to policies adhere to this same process. The only person who may merge a policy into the AspirePress Governance repository is the Executive Director of the AspirePress project, and they may only do so with Board approval.

### Compensation of Contributors

The Board of Directors may set a guideline for compensating project contributors in line with the goal of reducing the overall profitability of AspirePress and ensuring that AspirePress fulfills its mission to remain as close to revenue neutral as possible.

The policy for compensating contributors must take into account each contributor's contributions and equitably distribute any funds based on those contributions. Board of Directors members may be considered contributors for the purposes of this distribution. The distribution protocol must be in place no later than January 15th of the year in which it is effective; funds are distributed by December 31 of the same year.

Even when a policy for compensating contributors is established, the Executive Director shall have the final say in determining who qualifies under the policy and distributing the compensation. The Board of Directors may override any decision of the Executive Director with a unanimous vote of all Board members; the decision of the Board of Directors must align with the principles laid out in the annual compensation plan.

## The Executive Director

The Executive Director of AspirePress is selected and retained by the decision and vote of the Board of Directors. Their term runs from January 1 to December 31, and they may be reappointed by the Board of Directors for a new term each year.

### Powers of the Executive Director

The Executive Director is responsible for running the day-to-day operations of AspirePress. They may conduct the following operations:

* Spend money on behalf of AspirePress.
* Negotiate contracts and partnerships, subject to approval.
* Manage contributors, members and the project at large.
* Set the roadmap, vision and goals of the project.
* Establish timelines and deadlines for various aspects of the project.
* Set pricing for any products that AspirePress is to sell.
* Establish agreements for the inflow of revenues to AspirePress with outside organizations.
* Represent AspirePress in the community.
* Maintain access to all AspirePress accounts.
* Approve or reject new policies.
* Nominate Board of Directors members and set the size of the Board of Directors.
* Propose policies, compensation structures, and other governance model concepts.
* Maintaining communications, finance and legal for AspirePress.
* Day-to-day operations as required.

### Fiduciary responsibility

The Executive Director shall be required and expected to act in the best interests of AspirePress at all times. Failure to do so is grounds for removal. 

### Nomination, Term and Dismissal

An Executive Director shall be nominated by the Board of Directors for a one-year term, beginning February 1 and ending January 31 of the following year. The Board of Directors must approve the Executive Director’s selection by a majority vote.

The Executive Director may be dismissed by the Executive Committee during their term only for cause. The causes that they may be terminated for are:

* Failure to act as a fiduciary for AspirePress.
* Arrest or indictment of any crime of moral turpitude, including fraud, embezzlement, or theft.
* Conviction of any crime of moral turpitude.
* Violation of the Code of Conduct as determined by the independent Code of Conduct Review Board
* Failure to perform the duties and responsibilities of the Executive Director as enumerated.
* Failure to enforce and uphold the policies of AspirePress.

The Board of Directors may also elect not to renew an Executive Director at the conclusion of their term, for any reason. 

### Compensation

The Executive Director shall receive compensation of 1% of the net revenues of AspirePress, up to $100,000 per year. This reflects the fact that the Executive Director has an active, day-to-day responsibility to mange the project, which may or may not be a full-time job. They shall be paid monthly, based on the revenues of that month, and shall be compensated no less than $1,000 for their service in any given year.

### Responsibility for Board of Directors

The Executive Director of AspirePress is responsible for nominating Board of Director members and setting the size of the Board of Directors.

The size of the Board of Directors may only be changed effective at the start of the next Board of Directors term. The Executive Director may not grow the board such that they would create a new majority. The Board of Directors may only be made smaller by the number of individuals up for nomination in any given year. For example, if one seat must be nominated, then the Board of Directors may not shrink by more than one seat.

The Executive Director sits with the Board of Directors but does not vote unless there is a tie in the Board of Directors. 

### Responsibility for Policy

The Executive Director is responsible for implementing and upholding the policies of AspirePress. 

When a policy is approved by the Board of Directors, it must be approved by the Executive Director within one week. If the Executive Director disapproves the policy change, they must explain to the Board of Directors their reasoning. The Board of Directors may override the decision of the Executive Director with a 2/3rds vote (minimum three votes).

The Executive Director shall be responsible for publishing any policies in a public place. 

### Advice of the Board

The Executive Director may, at any time, request the advice of the Board of Directors on any matter. While the Board of Directors advice is not binding, the Executive Director should adhere to the recommendation unless there exists good cause to deviate.

### Selection of Officers

The Executive Director shall be empowered to select officers for AspirePress. They may create additional officers with specific roles at any time, subject to a majority approval of the Board of Directors. 

The officers that exist at this time are:

* Communications Director
* Finance Director
* Legal Director
* Project Manager
* Webmaster

If the Executive Director does not fill these roles, it is still their responsibility to conduct the tasks that fall under these roles.

### Accounting and Money

The Executive Director shall be a signer on bank accounts for AspirePress and have the obligation to meet the expenses of AspirePress. AspirePress may not incur debt for more than three months. The Executive Director shall make a monthly financial report for AspirePress available to the Board of Directors no later than fifteen days after the close of the last month. An annual report shall be provided no later than 30 days from the close of the fiscal year.

The Executive Director is responsible for the accounting of AspirePress. This may be delegated to a third party hired by the Executive Director. 

The Executive Director is responsible for preparation of any tax returns or government documents required by law. The Executive Director may outsource this task to another party. If this is outsourced, it must be to a CPA or other legally qualified, enrolled agent.

### Legal Compliance

The Executive Director shall be responsible for any and all legal compliance for AspirePress. They are obligated to seek counsel from a lawyer when needed, and may not provide legal advice or rely upon services like LegalZoom for any action taken by AspirePress.

## Legal Formation

AspirePress is legally a for-profit LLC. However, AspirePress seeks to operate as close to revenue-neutral as possible in any given year. 

### LLC Member

Every LLC has at least one member. The member of the AspirePress LLC shall be responsible for government compliance and filing. The member may be part of the Board of Directors or serve as the Executive Director, but they are not required to be in either role.

The member shall sign a contract with each member of the Board of Directors and the Executive Director empowering them to operate AspirePress based on the terms of this document. 

#### Emergency Powers

The member is responsible for the operation of the LLC in a legally compliant way. If the Board of Directors or the Executive Director violate the law or act in a manner contrary to this document or good sense, the member may suspend one or both for up to one week, pending a vote of the membership. The membership shall vote on whether the suspension should be permanent, at which point the member shall select new individuals to fill the seats vacated.

If the membership does not approve of the member’s suspension of the individuals, they shall be reinstated at the conclusion of the vote. The member may not attempt to suspend any previously suspended person for fourteen days following the vote.

During a suspension, all operations of AspirePress are paused, except mandatory filings, payment of obligations previously incurred, and legal compliance.

### Conversion to other legal forms

The member may propose conversion to other legal forms if and when appropriate. The Board of Directors and Executive Director shall be responsible for approving such a change.

### Bank accounts and finance

The member shall be the owner on all accounts of AspirePress and shall receive a financial report as if they were a member of the Board of Directors. The member shall not have the authority to add signers, issue cards or other payment instruments, or spend money unless they are also the Executive Director of AspirePress. However, they may spend money to meet legal obligations and as required by the government. Please review, @toderash @chuckadams @asirota @afragen @antpb @mattleach89 @namithj 

### Ability to serve

The member may serve as Executive Director or as a member of the Board of Directors, as determined appropriate by the parties responsible for those selections.

## Membership in AspirePress

Any individual who contributes code, documentation or other assets to AspirePress in the previous twelve months shall be deemed a member for the purposes of voting and participation. The Board of Directors may, by majority vote, consider other individuals members for the purposes of participation in the project. However, a member selected through this process must be a member for six months before they are eligible to serve in a role.

If a member ceases to be a member of AspirePress and holds a role within AspirePress, they may continue to serve for six months. Once they have rectified their member status, they may continue until the end of their term. 

The Board of Directors shall have the authority to develop procedures and policies that determine membership more clearly.

## Amendments

This document may be amended by following the policy process. However, the approval process requires a 3/4ths vote (minimum three votes) to propose an amendment, and a 2/3rds vote of the membership with a minimum of 50% of members voting. 

Nothing approved as an amendment to this document shall change the rights and responsibilities of the member.

**Comment by sarah-savage** [comment]
Please review, @chuckadams @afragen @antpb @asirota @mattleach89 @namithj @toderash

**Reply by mattleach89** [reply]
Reads well @sarah-savage. Really good writeup that covers pretty much everything I could think of. Fine with the Bank Account writeup, given that has specifically been highlighted for review.

Brain bump below. Sorry its a scattergun!

Perhaps first action for the board is to ensure a transparent method of communication exists for key stakeholders & the mechanics for members. IE: How can they vote, how can we demonstrate it is being heard. This creates a communication framework we need to onboard internally at AP, but also outwardly on channels like X, Reddit, HN etc

WP core devs/contributors, community feedback channels etc - should be clear on how we speak to those too, given we wouldn't want to be perceived as a voice for 'Commercial Entity A' over 'Commercial Entity B'. This could weigh into who could be elected too?

I will now spend Christmas eating food & thinking about governance. Sure the kids will love that.

**Comment by namithj** [comment]
Well thought out and comprehensive.

**Comment by chuckadams** [comment]
I like it very much: we'll need operational details like how the voting process actually happens (especially when there's a community-wide vote), but the overall structure works for me.   We'll definitely want to hammer this out further in the upcoming governance talks, but now we have a concrete proposal to start with.

**Comment by toderash** [comment]
Well done, lots of work here and quite comprehensive! Rather than dig into any typos and wordings in the first draft, I'll focus on procedural / policy comments so we can discuss conceptually. I'll use the same headings for context.

## Board of Directors

* 3-7 members might be a bit small long-term, but at the outset it's a good size. Consider an expansion clause if revenue from all sources exceeds $x, the board is increased to 7-12 members (or whatever) and must then include someone with a financial background. A larger board can also stipulate geographic or other forms representation that aren't as feasible with a smaller board.
* partial replacement every year is good, this is all good stuff
* vacant seats should be allowed for up to 6 months at the end of a term or up to 3 months at any other time, at the board's discretion (i.e., 2.5 years into a 3-year term, the board member was already due to be replaced). Also stipulate that a mid-term replacement inherits the term rather than beginning a new 3-year one so they remain staggered.
* a leave of absence of up to 6 months is not a vacant seat; any longer maybe needs an interim/proxy or replacement
* Other board stipulations could be no more than _n_ members (or %) from any one corporation, including subsidiaries and related parties
* Nomination process could be broadened to allow nominees not by the ED, who could essentially gatekeep the board members. Board members could also be allowed to nominate, and could also take nominations from the community through some process, maybe needing x signatures or something.

### Board Enumerated Powers

* contracts = contracts & expenses? Could set the limit at $1,000 or x% of budget, whichever is greater. This needs to dovetail with the ED's powers; i.e., they have discretion below the threshold and require approval above it.

### Compensation of the Board of Directors

* I might tend to be a bit more vague here, but word it so that they can be reimbursed for expenses plus paid a stipend to a collective total of not more than x% of Gross Profit or y% of EBITDA Net Profit or $z, whichever is greatest. Or something. iow, stipends are paid after all other expenses, and only up to a limit.
* Changes to compensation structures to be made based on recommendation of an independent advisory committee struck for the purpose at the board's initiation.

### Compensation of Contributors

* Perhaps stipulate that unless the contributor is an employee or contractor to AspirePress, all compensation is considered an honorarium. (Basically don't let it become subject to employment law.)
* ED & Board both have a form of veto here, which is a little odd. Could just say that for this purpose, the ED also participates in the decision with a vote.

### Powers of the Executive Director

* authorized spending limits to dovetail with board responsibilities above
* set vision and goals is usually board territory, with roadmap falling to ED for how to get there. This ensures a single individual can't change it, but they can make recommendations to the board to adopt
* Approve or reject new policies should have some scope - some of the policy ones might be board decisions, while SOPs would be the ED's focus. "Propose policies" is another bullet point in the draft, but relates here.
* nominate board and set its size allows too much control over the board. Board needs to set its own size, and nomination needs to include but go beyond the ED (noted above)

### Nomination, Term and Dismissal [of ED]

* maybe 2/3 majority of the board? It's the key role.

### Compensation

* This section subject to update or revision by the board of directors.

### Responsibility for Board of Directors && for Policy

* See notes above for limitation/distribution of this role

### Selection of Officers

* Webmaster would fall under Communications Director (not a technical role) but we may want an IT or IS Director
* Any individual director may be assigned multiple roles within legal guidelines (e.g. in some jurisdictions where 2 signatures are required from specific officers for certain cases, they need to be 2 different people, so "President & CEO" can't also be the secretary or treasurer)
* Officers may also be asked from time to time to serve as an advisory panel to the board of directors.

## Legal Formation

* I'll go through this a bit later - some of it is going to be jurisdiction-specific, but will thin through the general tone and the who's-a-member parts. Basically it ends up being the rules for how we make a for-profit corporation function in the way we want as a for-benefit or NFP corp.



# Bottom Line

Excellent work here, @sarah-savage , thanks for kicking this off! I'm not committed on any of the above, just meaning to raise some discussion points. We might be on the same page and just need to clarify, or my note might be addressed in some other way already, so just through them out for discussion and holding them loosely.

**Reply by asirota** [reply]
> * 3-7 members might be a bit small long-term, but at the outset it's a good size. Consider an expansion clause if revenue from all sources exceeds $x, the board is increased to 7-12 members (or whatever) and must then include someone with a financial background. A larger board can also stipulate geographic or other forms representation that aren't as feasible with a smaller board.

The board is not the administrative arm of the organization. It is a governance arm. Any financial management will most likely need to be done by a volunteer or paid financial manager/accountant/bookkeeper. They should not be part of the board.

**Comment by asirota** [comment]
Extraordinary start @sarah-savage 

Re board member compensation. 

This is not a standard policy to compensate volunteer board directors. In fact it may be in conflict in the jurisdiction of incorporation. 

I'd suggest an alternative model:


In accordance with the corporation’s bylaws, directors may receive an honorarium for their services, subject to the following conditions:
	1.	Honorarium Request: Directors may submit a request for an honorarium.
	2.	Maximum Amount: The honorarium shall not exceed $2,500 within any given fiscal year. (Covers a conference trip say or another incidental expenses.)
	3.	Approval Process: The disbursement of honorariums requires approval by the board of directors through a majority vote.

----

This provision ensures that any compensation to directors is controlled and transparent, aligning with best practices for nonprofit governance. ￼

It’s important to note that, in Ontario, under the Ontario Not-for-Profit Corporations Act (ONCA), directors are generally expected to serve without remuneration, except for reimbursement of reasonable expenses incurred in the performance of their duties. This is probably the case in Maryland too. 

Therefore, any board member compensation or honorarium should be carefully considered to ensure compliance with the jurisdiction of the organization and its bylaws.

**Reply by toderash** [reply]
Agreed. This is generally why the term "honorarium" is used as it disconnects the $ with the value or amount of effort contributed so it's clearly distanced from remuneration which would normally be subject to employment law.

**Comment by sc0ttkclark** [comment]
~~In my reading of this as it is, I see that AspirePress would need at minimum $4167 a month to cover the compensation for the board alone.

Should that be better ramped up according to revenue and if revenue ever dips then how should that impact compensation?~~

**Reply by sc0ttkclark** [reply]
Never mind, just read the 1% part!

**Reply by asirota** [reply]
If we decided on a yearly honorarium of say $2,500 per year, 5 board members would require a dotal of $12,500 per year (about $1000 per month) to honour the board. I believe this is quite reasonable and can scale as funding ramps up and/or the number of board members needs to be increased. Starting at anything more than 5 I don't think is necessary right now.

**Reply by toderash** [reply]
I like the concept of a maximum percentage subject to a maximum of net, which prevents causing a deficit from the payout. The maximum of net could be a higher percentage or could be set to maintain a minimum amount of operating capital. The other mechanism I like would be to have the board able to vote on compensation but not recommend it - recommendations for board compensation would then come from a group outside of the board.
